File: 2/3.py
import math

# eps = float(input("Введите точность eps: "))
eps = 0.0001
x = float(input("Введите x: "))

n = 1
z = 1
s = z

while abs(z) > eps:
    z = (-1) ** n * x ** (2 * n) / (math.factorial(2 * n))
    s += z
    n += 1
print("\tСумма равна: ", s)

print("\tCos(x):      ", math.cos(x))

File: 2/3.pas
var
n:integer;
z,sum,x,eps:real;
begin
  eps:=0.0001;
  z:=1;
  n:=1;
  sum:=1;
  writeln('Введите x:');
  read(x);
  writeln('Заданная точность: 0.0001');
  
  repeat
    z:=z*(-power(x,2)/(2*n*(2*n-1))); //реккурентная формула: из z(n-1) => находим z(n)
    sum:=sum+z;
    n+=1;
  until abs(z)<eps;
  
  write('Результат: ');
  writeln(sum);
  
  writeln('Cos(x): ',  cos(x));
end.